src/pages/MyPage/MyPage.js

```diff
@@ -1,7 +1,399 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useEffect } from 'react';
+import MyPageDropdown from './MyPageDropdown/MyPageDropdown';
+import MyPageContent from './MyPageContent/MyPageContent';
 
 const MyPage = () => {
-  return <div>MyPage</div>;
+  const [result, setResult] = useState([]);
+
+  const [myPagedropdownMenu, setMyPagedropdownMenu] = useState(false);
+
+  const closeDropdownMenu = () => {
+    setMyPagedropdownMenu(false);
+  };
+
+  useEffect(() => {
+    fetch(`./data/mock.json`, { method: 'GET' })
+      .then(res => res.json())
+      .then(result => setResult(result));
+  }, []);
+
+  return (
+    <>
+      <MyPageNav>
+        {NAV_LIST.map(list => {
+          return <NavList key={list}>{list.name}</NavList>;
+        })}
+      </MyPageNav>
+      <MyPageTop>
+        {LIST_TOP.map(data => {
+          return <ListTop key={data}>{data.name}</ListTop>;
+        })}
+      </MyPageTop>
+      <MyPageBox>
+        <MyPageContents>
+          <MyPageUserWrap>
+            <MyPageUserCont>
+              <MyPageUser>
+                <MyPageSocial
+                  src="./images/share.png"
+                  onClick={() => {
+                    setMyPagedropdownMenu(prev => !prev);
+                  }}
+                  // closeDropdownMenu={closeDropdownMenu}
+                />
+                {myPagedropdownMenu && (
+                  <MyPageDropdown closeDropdownMenu={closeDropdownMenu} />
+                )}
+                <MyPageUserProfile>
+                  <MyPageProfileImg src="./images/user.png" />
+                </MyPageUserProfile>
+                <MyPageUserId>{result.userId}</MyPageUserId>
+                <MyPageFollowWrap>
+                  <MyPageFollowBox>
+                    <MyPageFollow>팔로워</MyPageFollow>
```

Comment:
styled component는 유니크한 className을 생성해주기 때문에 컴포넌트명을 전부 붙여주지 않으셔도 됩니다!
```suggestion
                <UserProfile>
                  <ProfileImg src="./images/user.png" />
                </UserProfile>
                <UserId>{result.userId}</UserId>
                <FollowWrap>
                  <FollowBox>
                    <Follow>팔로워</Follow>
```